### DISCUSSION TITLE
Unpaid not-for-profit enterprise tier

### DISCUSSION THREAD

**Discussion by almereyda** [outline/outline#7123] [Ideas]
In

- https://github.com/outline/outline/issues/5078#issuecomment-1957697568

I had mentioned a distribution praxis by GitLab, who offer certain paid features to free self-hosters, when they are willing to share anonymous usage data.

The comment about that GitLab praxis should have been posed in form of a direct question:

Could you [@tommoor] imagine introducing a medium, non-paid tier for not-for-profits, who are open to sharing anonymous usage data and receive a selected blend of enterprise features?

**Comment by tommoor** [comment]
Very unlikely, we already offer the community edition and 30% discount for non-profits on all licenses – honestly more is being given away already than makes any business sense 😉 

Usage data is not really of any value in this case.

**Comment by almereyda** [comment]
Good to know, yes. Thanks for the answer.
